src/pages/dar_application/ResearcherInfo.js

```diff
@@ -84,14 +85,15 @@ export default function ResearcherInfo(props) {
 
         div({className: 'dar-application-row'}, [
           h3('1.2 Researcher Identification' + (formData.checkCollaborator ? ' (optional)' : '')),
-          span({ className: `${showNihValidationError ? 'errored' : 'default-color'}`, isRendered: formData.checkCollaborator !== true }, [
+          !readOnlyMode && span({ className: `${showNihValidationError ? 'errored' : 'default-color'}`, isRendered: formData.checkCollaborator !== true }, [
```

Comment:
In this file, there is an Alert on line 62 that needs similar `readOnly` treatment. We don't need to show that alert to DAC members for users who haven't authenticated with eRA Commons:

![Screenshot 2023-10-18 at 10 10 38 AM](https://github.com/DataBiosphere/duos-ui/assets/116679/0a3ddb55-19fa-41f1-98f1-b032b9371735)